src/loader.js

```diff
@@ -115,6 +118,37 @@ function setupModuleLoader(window) {
           _runBlocks: runBlocks,
 
           /**
+           * @ngdoc method
+           * @name angular.Module#info
+           * @module ng
+           *
+           * @param {Object=} info Information about the module
+           * @returns {Object|Module} The current info object for this module if called as a getter,
+           *                          or `this` if called as a setter.
+           *
+           * @description
+           * Additional info about this module
+           * For example you could put the version of the module in here.
+           *
+           * ```js
+           * angular.module('myModule', []).info({ version: '1.0.0' });
+           * ```
+           *
+           * The global method `angular.info()` can be used to retrieve this info:
+           *
+           * ```js
+           * var version = angular.info('myModule').version;
+           * ```
+           */
+          info: function(value) {
+            if (isDefined(value)) {
+              info = value;
```

Comment:
@ThomasBurleson - I would rather not use `extend` here. Can you give a concrete use case?